models/BaseProxy.cfc

```diff
@@ -213,4 +213,43 @@ component accessors="true" {
 		return ( findNoCase( "ForkJoinPool", getThreadName() ) NEQ 0 );
 	}
 
+	void function sendExceptionToLogBoxIfAvailable( required any exception ){
```

Comment:
The problem I have with these methods is that cbproxies doesn't care if it's coldbox or wirebox standalone. This is very ColdBox centric.Basically this won't work with WireBox standalone.

Reply:
I can see that for the LogBox one.  Does WireBox have a way to get LogBox?

For the interceptor, that has to be ColdBox, so no problem there.